packages/kbn-i18n/GUIDELINE.md

```diff
@@ -372,6 +396,82 @@ Here is an example of message translation depending on a plural category:
 
 When `conflictFieldsLength` equals 1, the result string will be `"A field is defined as several types (string, integer, etc) across the indices that match this pattern."`. In cases when `conflictFieldsLength` has value of 2 or more, the result string - `"2 fields are defined as several types (string, integer, etc) across the indices that match this pattern."`.
 
+### Text with markdown
+
+There is some support for using markdown and you can use any of the following syntax:
+
+#### Headers
+
+  ```js
```

Comment:
These should be `md` instead of `js`.we can also get rid of the padding before the triple backticks and inside the code block
```suggestion
```md
```